dockerfiles/notebook/Dockerfile

```
@@ -14,6 +21,7 @@ COPY Pipfile.lock .
 
 RUN pip install --no-cache-dir pipenv==2021.5.29 \
     && pipenv install --system \
-    && rm -rf Pipfile* /tmp/* /var/tmp/*
+    && rm -rf Pipfile* \
+    && Rscript -e "install.packages(c('DBI', 'RPostgreSQL', 'themis'), dependencies = TRUE, error = TRUE, repos='https://cran.microsoft.com/snapshot/2022-12-16/')"
```

```suggestion
    && Rscript -e "install.packages(c('DBI', 'RPostgreSQL', 'themis'), dependencies = TRUE, error = TRUE, repos='https://cran.microsoft.com/snapshot/2022-12-16/')"
    && rm -rf Pipfile* /tmp/* /var/tmp/*
```

masterのL.17に関してはmamba移行とは関係ない差分 ( https://github.com/The-Japan-DataScientist-Society/100knocks-preprocess/pull/162 )、かつ、tmpファイルの削除はRパッケージインストール後に行った方が良いと考えられるため、上記のようにしていただけるとありがたいです。

---

上記のように修正しました！